<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginDialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎使用</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_more">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>680</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>︾</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>381</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lE_Password">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>456</width>
        <height>61</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入密码</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="cB_ListUser">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>61</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>自定义的用户</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>247</y>
     <width>381</width>
     <height>420</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pB_1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pB_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pB_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pB_4">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pB_5">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pB_6">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pB_7">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pB_8">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pB_9">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pB_Login">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>禅意雪松体</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pB_0">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">numButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pB_Exit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>禅意雪松体</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 127);</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>710</y>
     <width>341</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="cB_style">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>默认风格</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Iron Man</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大话西游</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>千岛湖</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cB_turnOn">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>开机自启动</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cB_turnOff">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>退出时关机</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="numButtonGroup"/>
 </buttongroups>
</ui>
